frontend/src/app/[locale]/process/ProcessNext.tsx

```diff
@@ -0,0 +1,114 @@
+import { ExternalRoutes } from "src/constants/routes";
+
+import { useMessages, useTranslations } from "next-intl";
+import Link from "next/link";
+import React from "react";
+import {
+  Grid,
+  GridContainer,
+  IconList,
+  IconListContent,
+  IconListIcon,
+  IconListItem,
+} from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+const ProcessNext = () => {
+  const t = useTranslations("Process.next");
+  const messages = useMessages() as unknown as IntlMessages;
+
+  const keys = Object.keys(messages.Process.next.list);
+
+  return (
+    <GridContainer className="grid-container padding-y-1 tablet:padding-y-3 desktop-lg:padding-y-6 padding-top-0 tablet:padding-top-0 desktop-lg:padding-top-0">
+      <h2 className="margin-bottom-0 tablet-lg:font-sans-l desktop-lg:font-sans-xl margin-top-0">
+        {t("title")}
+      </h2>
+      <Grid row gap>
+        <Grid className="tablet:grid-col">
+          <IconList className="usa-icon-list--size-md grid-row">
+            {keys.map((key) => {
+              const title = t(`list.${key}.title`);
+              const content = t.rich(`list.${key}.content`, {
+                p: (chunks) => (
+                  <p className="font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+                    {chunks}
+                  </p>
+                ),
+                linkGithub3045: (chunks) => (
```

Comment:
as per my comment above I think this could benefit from one level of abstraction so we can avoid repeating ourselves quite so much here

```
const gitHubIssueLink = (issueNumber) => (chunks) => (
       <Link
                    target="_blank"
                    className="usa-link--external"
                    href={`${ExternalRoutes.GITHUB_REPO}/issues/${issueNumber}`}
                  >
                    {chunks}
                  </Link>
                ),
);
...

linkGithub3045: gitHubIssueLink(3045)
```